src/mots/config.py

```diff
@@ -150,7 +150,7 @@ def clean(file_config: FileConfig, write: bool = True):
         file_config.write()
 
 
-def validate(config: dict, repo_path: str):
+def validate(config: dict, repo_path: str, do_raise: bool = True):
```

Comment:
What's the point of `do_raise`?

Reply:
This method is used both in the CLI (where exceptions are raised), and when integrated with Lando (or other Python things), in which case it is more useful to return a list of errors instead of raising an exception. FWIW, the general validation and raising of exceptions needs to be improved, and is partially tracked in [bug 1753716](https://bugzilla.mozilla.org/show_bug.cgi?id=1753716).